civo/loadbalancer/resource_loadbalancer.go

```diff
@@ -0,0 +1,530 @@
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Name of the load balancer.",
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+			"service_name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Name of the service associated with the load balancer.",
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+			"network_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Network ID associated with the load balancer.",
+				ValidateFunc: utils.ValidateUUID,
+			},
+			"algorithm": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "round_robin",
+				Description:  "Load balancing algorithm, either 'round_robin' or 'least_connections'.",
+				ValidateFunc: validation.StringInSlice([]string{"round_robin", "least_connections"}, false),
+			},
+			"external_traffic_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "External traffic policy, either 'Cluster' or 'Local'.",
+				ValidateFunc: validation.StringInSlice([]string{"Cluster", "Local"}, false),
+			},
+			"session_affinity": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Session affinity setting, either 'ClientIP' or 'None'.",
+				ValidateFunc: validation.StringInSlice([]string{"ClientIP", "None"}, false),
+			},
+			"session_affinity_config_timeout": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Description:  "Timeout for session affinity in seconds.",
+				ValidateFunc: validation.IntAtLeast(0),
+			},
+			"enable_proxy_protocol": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Enable proxy protocol, options are '', 'send-proxy', 'send-proxy-v2'.",
+				ValidateFunc: validation.StringInSlice([]string{"", "send-proxy", "send-proxy-v2"}, false),
+			},
+			"max_concurrent_requests": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Maximum concurrent requests the load balancer can handle.",
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+			"cluster_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "ID of the cluster associated with the load balancer.",
+				ValidateFunc: utils.ValidateUUID,
+			},
+			"firewall_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "ID of the firewall associated with the load balancer.",
+				ValidateFunc: utils.ValidateUUID,
+			},
+			"firewall_rule": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Firewall rules for the load balancer (e.g., 'all', '80,443', '40-80,90-120').",
+			},
+			"backend": {
```

Comment:
This should be `backends`

Reply:
It is supposed to look like

```
resource "civo_load_balancer" "example" {
  name                     = "uzaiii-lodbal"
#   service_name             = "example-service"
  network_id               = civo_network.example.id
  algorithm                = "round_robin"
  external_traffic_policy  = "Cluster"
  session_affinity         = "ClientIP"
  session_affinity_config_timeout = 120
  enable_proxy_protocol    = "send-proxy-v2"
  max_concurrent_requests  = 500
  firewall_id              = civo_firewall.example.id

  backend {
    ip             = "192.168.1.11"
    protocol       = "TCP"
    source_port    = 443
    target_port    = 8443
    health_check_port = 9443
  }
  backend {
    ip             = "192.168.1.10"
    protocol       = "TCP"
    source_port    = 80
    target_port    = 8080
    health_check_port = 9090
  }
}
```

so since we define each backend separately so imo it should be `backned`, if we define all of them in a single array then it should be `backends`